src/stores/tenant.rs

```diff
@@ -314,7 +318,7 @@ impl TenantStore for PgPool {
     #[instrument(skip(self))]
     async fn update_tenant(&self, id: &str, params: TenantUpdateParams) -> Result<Tenant> {
         let res = sqlx::query_as::<sqlx::postgres::Postgres, Tenant>(
-            "UPDATE public.tenants SET id = $2 WHERE id = $1 RETURNING *;",
+            "UPDATE public.tenants SET id = $2, updated_at = NOW() WHERE id = $1 RETURNING *;",
```

Comment:
What is the purpose even of this function? It looks like this is never called anyway. Should just remove as `id` shouldn't be updated.

Reply:
It was added a long ago, probably the author expect to use it somehow. I'll remove this rudiment.